tools/github_webhook_processor.php

```diff
@@ -146,15 +157,11 @@ function tag_pr($payload, $opened) {
 	else if ($mergable === FALSE)
 		$tags[] = 'Merge Conflict';
 
-	if(has_tree_been_edited($payload, '_maps'))
-		$tags[] = 'Map Edit';
-	else
-		$remove[] = 'Map Edit';
-
-	if(has_tree_been_edited($payload, 'tools'))
-		$tags[] = 'Tools';
-	else
-		$remove[] = 'Tools';
+	autotag_tree($payload, '_maps', 'Map Edit', $tags, $remove);
```

Comment:
Isn't there a way to make this a dictionary or something so there's no copypasta?